docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2807,6 +2807,349 @@ have not been included. These are:
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 1%
+    - N/A
+    - Incorrect Select
+    - Noise for all types of addresses will work in the same way 
+  * - Zip Code 
+    - Census, Household Surveys, WIC, Taxes (both) 
+    - 1%
+    - 1
+    - Numeric miswriting
+    - Applies to home, mailing, and employer addresses 
+  * - Relationship to head of household 
+    - Census 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Sex 
+    - Census, Household Surveys, WIC 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Race/Ethnicity 
+    - Census, WIC
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - SSN
+    - Taxes (both), SSA
+    - 1%
+    - 1
+    - "Borrowed" SSN, Copy from within Household, Numeric miswriting 
+    - Note that not all types of noise apply to all observers, details below 
+  * - ITIN
+    - Taxes 1040
+    - 1%
+    - 1
+    - Copy from within Household, Numeric miswriting 
+    - Note that not all types of noise apply to all observers 
+  * - Income / Wages
+    - Taxes (both)
+    - 1%
+    - 1
+    - Numeric miswriting
+    - Note that wages and income are on separate tax forms and noise is applied to each separately 
+  * - Employer ID 
+    - Taxes (both)
+    - 1%
+    - 1
+    - Numeric miswriting
+    - 
+  * - Employer Name 
+    - Taxes (both)
+    - 1%
+    - 1
+    - OCR, typographic 
+    - 
+  * - Type of Tax Form  
+    - Taxes (both)
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Type of SSA Event 
+    - SSA 
+    - 1%
+    - N/A
+    - Incorrect select 
+    - 
+  * - Date of SSA Event 
+    - SSA 
+    - 1%
+    - N/A
+    - Numeric miswriting, month and day swap 
+    - 
+
+The below section further describes types of noise including any code 
+available and information for implementation. **Software engineering team - please alert the research team if any of the below looks to be particularly challenging for further discussion.**
+
+**Converting from Noise Inputs to Noise Function Parameters for OCR, Phonetic, Typographic and Numeric Miswriting** 
+
+For simplicity for the end user, they will input two values for noise level: 
+
+#. Rows with Errors 
+#. Errors per Row 
+
+For types of noise others than those listed above, the only parameter is rows with error 
+and can be followed exactly. For noise types above, the functions we will use take different parameters: 
+
+#. Rows Eligible for Errors 
+#. Error Rate per Character OR per Token 
+
+These parameters, while related are different in key ways and therefore we need to 
+provide a way to convert between them. 
+
+Starting with #1 from both lists, we will calculate the number of rows eligible 
+for errors to approximate the user inputted rows with errors given. To calculate 
+this we can: 
+
+#. Compute the average length of strings in the column 
+#. Probability of at least 1 error = 1 - ((1 - (user defined errors per row / average string length))^(average string length))
```

Comment:
This is the point where I think this math is incorrect: (user defined errors per row / average string length) is not the character/token error rate that leads to the user-defined number of errors per row (whether that is per row overall or per row with at least one error), because this rate is applied to characters in _eligible_ rows.